### ISSUE TITLE
Add a checkbox so attendees can indicate whether they expect to need the bus between the station & the site

### ISSUE THREAD

**Issue by SamLR** [emfcamp/Website#1428]
We'd like to have a better idea of how many people will want to use the bus.

This doesn't need to be detailed, just general "Yes I'll want it". 

It should not block the ticket buying process might be worth gathering on the diversity page.

**Comment by jellybob** [comment]
This should probably be slightly more detailed - not just "I plan to use the bus", but "I plan to use the bus on [day] [morning/afternoon/evening]".

**Comment by Jonty** [comment]
I'd also pull this out into a separate thing from the diversity page (maybe just a step before) and we should probably also only show this to people who don't have a car ticket.

**Comment by SamLR** [comment]
> This should probably be slightly more detailed - not just "I plan to use the bus", but "I plan to use the bus on [day] [morning/afternoon/evening]".

I don't think this is super useful. Firstly a lot of people just wont fill it in (and each additional question reduces the response rate), secondly given the months between the event and most of the tickets going on sale a lot of people won't know. I think it's more useful to get a broad idea of how many people will want it and go from there.

I think we have a pretty good idea of /when/ folks are going to want to arrive/leave, and we can only move them in buckets of coach-size. The conversation that spurred this was do we want 2 or 3 coaches (with the note we have to hire them per day regardless) so any more fine-grained information than per-day seems overkill?

> I'd also pull this out into a separate thing from the diversity page (maybe just a step before) 

Possibly but I think if it's only a single checkbox (which I think it should be) it doesn't make sense to isolate it

> we should probably also only show this to people who don't have a car ticket.
good idea

**Comment by russss** [comment]
The reason why we need this is to try and work our how many people are leaving on the Sunday due to the track closure. We can already make a reasonable guess for how many people in total will be taking the train, but it helps significantly for our bus planning if we know approximately when they'll be arriving/departing.

**Comment by Jonty** [comment]
It _has_ to be more detailed to be useful - we need to know which day they plan to leave, and ideally if it'll be morning or afternoon.

**Comment by Jonty** [comment]
🫰

**Comment by SamLR** [comment]
Is the question then "if you planning to use the bus are you planning to leave on Sunday morning, Sunday afternoon or Monday morning?"
